es/index.html

```diff
@@ -43,7 +43,7 @@
         <a class="menu__item" href="/" itemprop='url'>
           <span itemprop="name">EN</span>
         </a>
-        <a class="menu__item" href="/es/" itemprop='url'>
+        <a class="menu__item" href="newsletter.macile.org/es" itemprop='url'>
```

Comment:
It's better to leave this as it was 😅  We want absolute internal linsk. If you use a full path, you want to use HTTP/HTTPS in front of it. 

We have to change the paths for the launch of the bulletin so I'm not going to ask that you fix now. 

Be careful about adding new work near the end of a PR revision unless requested. It is often better to point it out and dialogue with the reviewer. If it is small and valuable, they may ask you to make it. Otherwise, it gets handled in a follow-up PR.